pkg/proxy/proxy.go

```diff
@@ -143,3 +160,43 @@ func templateWithValue(templateString, value string) string {
 	}
 	return out.String()
 }
+
+func WithKubeRBACProxyParamsHandler(handler http.Handler, authzConfig *authz.Config) http.Handler {
```

Comment:
Hm, another filter. Strengthens my opinion to move it. WDYT?

Reply:
this one belongs close to the authorizer